packages/graph/hash_graph/lib/graph/src/api/rest/utoipa_typedef/subgraph/edges.rs

```diff
@@ -35,8 +56,8 @@ pub struct Edges {
     pub knowledge_graph: KnowledgeGraphRootedEdges,
 }
 
-impl From<crate::subgraph::edges::Edges> for Edges {
-    fn from(edges: crate::subgraph::edges::Edges) -> Self {
+impl Edges {
+    pub fn from_store_subgraph(edges: crate::subgraph::edges::Edges, vertices: &Vertices) -> Self {
```

Comment:
Naming of this function seems a bit weird if you don't actually pass in a `Subgraph`

Reply:
Yes, the name is not optimal, I renamed it b89f701f30178cf0dada4ff47c8b497489bf0c81
I'm still not super satisfied with the name, but as mentioned in the other comments, this will change anyway.